startup/10-motors.py

```diff
@@ -0,0 +1,76 @@
+from ophyd import Device, Component as Cpt, EpicsMotor
+
+
+# Temporary test motor
+class TestMotor(Device):
+    e   = Cpt(EpicsMotor, '-Ax:E}Mtr')
+    f   = Cpt(EpicsMotor, '-Ax:F}Mtr')
+    ctr = Cpt(EpicsMotor, '-Ax:XCtr}Mtr')
+    gap = Cpt(EpicsMotor, '-Ax:XGap}Mtr')
+
+
+class XYMotor(Device):
+    x = Cpt(EpicsMotor, '-Ax:X}Mtr')
+    y = Cpt(EpicsMotor, '-Ax:Y}Mtr')
+
+
+class Slits(Device):
+    b     = Cpt(EpicsMotor, '-Ax:B}Mtr')
+    i     = Cpt(EpicsMotor, '-Ax:I}Mtr')
+    o     = Cpt(EpicsMotor, '-Ax:O}Mtr')
+    t     = Cpt(EpicsMotor, '-Ax:T}Mtr')
+    x_ctr = Cpt(EpicsMotor, '-Ax:XCtr}Mtr')
+    x_gap = Cpt(EpicsMotor, '-Ax:XGap}Mtr')
+    y_ctr = Cpt(EpicsMotor, '-Ax:YCtr}Mtr')
+    y_gap = Cpt(EpicsMotor, '-Ax:YGap}Mtr')
+
+
+class DCM(Device):
+    # Virtual Motor
+    energy  = Cpt(EpicsMotor, '-Ax:E}Mtr')
+
+    # Real Motors
+    bragg   = Cpt(EpicsMotor, '-Ax:B}Mtr')
+    perp    = Cpt(EpicsMotor, '-Ax:Perp}Mtr')
+    para    = Cpt(EpicsMotor, '-Ax:Para}Mtr')
+    pitch   = Cpt(EpicsMotor, '-Ax:P}Mtr')
+    roll    = Cpt(EpicsMotor, '-Ax:R}Mtr')
+    yaw     = Cpt(EpicsMotor, '-Ax:Yaw}Mtr')
+    c2_bnd1 = Cpt(EpicsMotor, '-Ax:Bnd1}Mtr')
+    c2_bnd2 = Cpt(EpicsMotor, '-Ax:Bnd2}Mtr')
+    c1_bnd  = Cpt(EpicsMotor, '-Ax:Bnd}Mtr')
```

Comment:
This alignment is good for readability, but against PEP8 standards. Can we apply `black` formatting to the code, and also check with `flake8`? `isort` is used for some of the projects to sort the imports.

Reply:
Will do

Reply:
Thanks.